python/GafferVDBTest/VDBObjectTest.py

```diff
@@ -0,0 +1,45 @@
+import GafferTest
+
+import GafferVDB
+import IECore
+import GafferVDBTest
+import os
+
+class VDBObjectTest( GafferVDBTest.VDBTestCase ) :
+
+	def setUp( self ) :
+		GafferVDBTest.VDBTestCase.setUp( self )
+
+	def testCanLoadVDBFromFile( self ) :
+		sourcePath = os.path.join(self.dataDir, "utahteapot.vdb")
+		vdbObject = GafferVDB.VDBObject(sourcePath)
+
+		gridNames = vdbObject.gridNames()
+		self.assertEqual(gridNames, ['ls_utahteapot'])
+
+		metadata = vdbObject.metadata('ls_utahteapot')
+
+		expected = IECore.CompoundObject(
+			{
+				'name': IECore.StringData( 'ls_utahteapot' ),
+				'file_voxel_count': IECore.Int64Data( 6960047 ),
+				'file_bbox_min': IECore.V3iData( IECore.V3i( -500, -231, -308 ) ),
+				'file_bbox_max': IECore.V3iData( IECore.V3i( 480, 231, 308 ) ),
+				'is_local_space': IECore.BoolData( 0 ),
+				'is_saved_as_half_float': IECore.BoolData( 1 ),
+				'value_type': IECore.StringData( 'float' ),
+				'class': IECore.StringData( 'level set' ),
+				'file_mem_bytes': IECore.Int64Data( 10733944 ),
+				'vector_type': IECore.StringData( 'invariant' )
+			}
+		)
+
+		self.assertEqual(metadata, expected)
+
+
```

Comment:
All our other unit tests end with this so they can be run with `python file.py` :

```
if __name__ == "__main__":
	unittest.main()
```

I'm not sure how useful that really is, but I don't see any reason for the VDB tests to differ.